client/components/Registration-Page/register.css

```diff
@@ -0,0 +1,59 @@
+@import '../typography/typography.css';
```

Comment:
to keep our naming consistent, switch folder name to camelCase. Meaning `registrationPage`